src/settings/s_decisions.php

```diff
@@ -147,7 +147,80 @@ function render(SettingValues $sv) {
         "<p class=\"settingtext\">To collect <em>multiple</em> final versions, such as one in 9pt and one in 11pt, add “Alternate final version” options via <a href='", hoturl("settings", "group=opt"), "'>Settings &gt; Submission options</a>.</p>",
         "</div>\n\n";
     Ht::stash_script("foldup(\$\$('cbfinal_open'),null,{f:\"c\"})");
+
+	//Share with Open Academic
+    echo "<h3 class=\"settings g\">Share with search engines</h3>\n";
+    echo "<p><b>Click the button to share accepted paper meta data with search engines</b></p>";
+    echo Ht::submit("share", "Share with search engines");
+
+    //wakes up API
+    $api = 'https://openacademicapi.azurewebsites.net/api/papers';
+    $opts = array('http'=> array('method' => 'GET'));
+    $context = stream_context_create($opts);
+    $post_result = file_get_contents($api, false, $context);
+   
+    if (isset($_REQUEST["share"])) {
+        $this->share($sv, $api);
+    }
+
 }
+    //sends https post request to the api
+    function share(SettingValues $sv, $api) {
+
+        $share = array();
+        $share["provider"] = "HotCRP";
+        $share["setId"] = "";
+        $share["venue"] = $sv->conf-> long_name;
+        $share["shortVenue"] = $sv->conf-> short_name;
+        $share["confSite"] = $sv->conf-> opt["conferenceSite"];
+        $share["version"] = "1.0";
+    
+        $papers = array();
+        $result = $sv->conf-> q_raw("SELECT paperID, title, authorInformation, abstract FROM Paper WHERE outcome > 0");
+        while (($row = edb_row($result))) {
```

Comment:
In the recent commit, I changed it to enable users to search by using the decisionSelector and default to dec:yes. Please take a look!